src/main/java/com/tripply/booking/model/request/RoomBookingRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.tripply.booking.model.request;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class RoomBookingRequest {
+    @NotNull(message = "It can't be null")
```

Comment:
replace It with the variable name, if this type of message we will kept, the client will never know what validation failed.